dotcom-rendering/src/lib/useDeeplyReadTestVariant.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from 'react';
+import { useAB } from './useAB';
+
+type Variant =
+	| 'deeply-read-only'
+	| 'deeply-read-and-most-viewed'
+	| 'most-viewed-only'
+	| 'none';
+export const useDeeplyReadTestVariant = (): Variant => {
+	const [variant, setVariant] = useState<Variant>('none');
+	const ABTestAPI = useAB()?.api;
+	useEffect(() => {
+		if (
+			ABTestAPI?.isUserInVariant(
+				'DeeplyReadRightColumn',
+				'deeply-read-only',
+			)
+		) {
+			setVariant('deeply-read-only');
+		} else if (
+			ABTestAPI?.isUserInVariant(
+				'DeeplyReadRightColumn',
+				'deeply-read-and-most-viewed',
+			)
+		) {
+			setVariant('deeply-read-and-most-viewed');
+		} else if (
+			ABTestAPI?.isUserInVariant(
+				'DeeplyReadRightColumn',
+				'most-viewed-only',
+			)
+		) {
+			setVariant('most-viewed-only');
+		} else {
+			setVariant('none');
+		}
```

Comment:
The `AB` test api has a `runnableTest` method that will provide the current variant for a givent test, or `null` if the user is not in a test.


```ts
const AB = useAB();
const variantId = AB?.api.runnableTest(onwardJourneys)?.variant.id;
setVariant(variantId ?? 'none')
```